eos/form-factors/baryonic_TEST.cc

```diff
@@ -79,6 +80,96 @@ class BFvD2014FormFactorsTest :
         }
 } bfvd2014_form_factors_test;
 
+class DKMR2017FormFactorsTest :
+public TestCase
+{
+public:
+    DKMR2017FormFactorsTest() :
+    TestCase("dkmr2017_form_factors_test")
+    {
+    }
+
+    virtual void run() const
+    {
+        static const double eps = 1e-3;
+
+        Parameters p = Parameters::Defaults();
+        std::shared_ptr<FormFactors<OneHalfPlusToOneHalfPlus>> ff = FormFactorFactory<OneHalfPlusToOneHalfPlus>::create("Lambda_b->Lambda_c@DKMR2017", p);
+        // fix z-expansion parameters @DKMR2017 - all a_2's are zero by default
+        // Note that a0_perp_A = a0_long_A ve a0_perp_T5 = a0_long_T5
+        p["Lambda_b->Lambda_c::a_0_time^V@DKMR2017"]    =  0.33;
```

Comment:
Please also add the zeros for all second order coefficients explicitly to the test case.